src/ray/rpc/client_call.h

```diff
@@ -309,6 +326,9 @@ class ClientCallManager {
     }
   }
 
+  /// UUID of the cluster.
+  SafeClusterID cluster_id_;
```

Comment:
Add comment here why we need cluster_id to be thread safe. Where is the race conditions.